lib/matplotlib/backends/qt_editor/_formlayout.py

```diff
@@ -470,9 +469,10 @@ def update_buttons(self):
         for field in self.float_fields:
             if not is_edit_valid(field):
                 valid = False
-        for btn_type in (QtWidgets.QDialogButtonBox.Ok,
-                         QtWidgets.QDialogButtonBox.Apply):
-            btn = self.bbox.button(btn_type)
+        for btn_type in ["Ok", "Cancel"]:
```

Comment:
Old was `Ok`, `Apply`
```suggestion
        for btn_type in ["Ok", "Apply"]:
```